docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md

```diff
@@ -38,17 +40,17 @@ Devices similar to the ones we have used for benchmarking can be accessed using
 ## Features support matrix
 The table below demonstrates support of key features by OpenVINO device plugins.
 
-| Capability | CPU | [GPU](./GPU.md) | GNA | VPU |
-| ---------- | --- | --- | --- | --- |
-| [Heterogeneous execution](../hetero_execution.md)| Yes | Yes | ? | ? |
-| [Multi-device execution](../multi_device.md) | Yes | Yes | ? | ? |
-| [Automatic batching](../automatic_batching.md) | No | Yes | ? | ? |
-| [Multi-stream execution](@ref openvino_docs_optimization_guide_dldt_optimization_guide) | Yes | Yes | ? | ? |
-| [Models caching](../Model_caching_overview.md) | Yes | Partial | ? | ? |
-| [Dynamic shapes](../ov_dynamic_shapes.md) | Yes | Partial | ? | ? |
-| Import/Export | Yes | No | ? | ? |
-| [Preprocessing acceleration](../preprocessing_overview.md) | Yes | Yes | ? | ? |
-| [Stateful models](../network_state_intro.md) | Yes | No | ? | ? |
-| [Extensibility](@ref openvino_docs_Extensibility_UG_Intro) | Yes | Yes | ? | ? |
+| Capability | CPU | [GPU](./GPU.md) | GNA | VPU | [Arm&reg; CPU](ARM_CPU.md) |
+| ---------- | --- | --- | --- | --- | --- |
+| [Heterogeneous execution](../hetero_execution.md)| Yes | Yes | ? | ? | No |
+| [Multi-device execution](../multi_device.md) | Yes | Yes | ? | ? | No |
+| [Automatic batching](../automatic_batching.md) | No | Yes | ? | ? | No |
+| [Multi-stream execution](@ref openvino_docs_optimization_guide_dldt_optimization_guide) | Yes | Yes | ? | ? | Yes |
+| [Models caching](../Model_caching_overview.md) | Yes | Partial | ? | ? | No |
+| [Dynamic shapes](../ov_dynamic_shapes.md) | Yes | Partial | ? | ? | No |
+| Import/Export | Yes | No | ? | ? | No |
+| [Preprocessing acceleration](../preprocessing_overview.md) | Yes | Yes | ? | ? | No |
```

Comment:
do we support any of convert, transpose, interpolate, color space conversions? We probably need to use partial and give more explanations in the plugin page.